postgres/datadog_checks/postgres/statements.py

```diff
@@ -235,6 +234,11 @@ def _load_pg_stat_statements(self):
                     "Unable to collect statement metrics because pg_stat_statements is not installed "
                     "in this database"
                 )
+            elif isinstance(e, psycopg2.errors.UndefinedTable) and 'pg_stat_statements' in str(e.pgerror):
+                error_tag = "error:database-{}-pg_stat_statements_not_created".format(type(e).__name__)
```

Comment:
While we're here should we change the other error name to `pg_stat_statements_not_loaded`. The term "load" seems closer to what the actual action is "adding pg_stat_statements to shared_preload_libraries".